pkg/plugins/policies/meshtrafficpermission/api/v1alpha1/meshtrafficpermission.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+
+package kuma.plugins.policies.meshtrafficpermission.v1alpha1;
+
+import "mesh/options.proto";
+option go_package = "github.com/kumahq/kuma/pkg/plugins/policies/meshtrafficpermission/api/v1alpha1";
+
+import "common/v1alpha1/targetref.proto";
+import "config.proto";
+
+option (doc.config) = {
+  type : Policy,
+  name : "MeshTrafficPermission",
+  file_name : "meshtrafficpermission"
+};
+
+// MeshTrafficPermission defines permission for traffic between data planes
+// proxies.
+message MeshTrafficPermission {
+  option (kuma.mesh.policy) = {
+    // Toggle this to have the policy registered or not in Kuma
+    skip_registration : false,
```

Comment:
In you PR description you say that you have this to `true` but in fact it's false, what should this be for new policies? @lobkovilya